tables/ba.utb

```diff
@@ -0,0 +1,57 @@
+#-index-name: Bashkir
+#-display-name: Bashkir braille
+
+#+locale: ba
+#+type: literary
+
+# -----------------
+# TODO: Please add a reference to official documentation about
+# the implemented braille code. Preferably submit the documents
+# to https://github.com/liblouis/braille-specs.
+# -----------------
+
+#-license: LGPLv2.1
+
+#  Copyright (C) 2020 Rustam churagulov <tcdl@.ru>
```

Comment:
Something went wrong with this e-mail address?

Reply:
yes, it was me who accidentally indicated the mailing address incorrectly: tcdl@yandex.ru